build.sh

```diff
@@ -240,6 +247,7 @@ make install
 echo "======= Install NEST end ======="
 
 echo "======= Test NEST start ======="
+export PYTHONPATH=$HOME/.cache/csa.install/lib/python2.7/site-packages:$PYTHONPATH
```

Comment:
w-klijn, indeed, but this has already been reworked and improved in the latest version.